packages/engine-core/src/binary/Connection.ts

```diff
@@ -88,7 +91,7 @@ export class Connection {
         ...headers,
       },
       body,
-      bodyTimeout: 0,
+      // bodyTimeout: 0,
```

Comment:
??

Reply:
I think this option does not exist anymore, to check if it was moved somewhere else and fix it

Reply:
It looks like this should still work
https://github.com/nodejs/undici/blob/ffc6341559ae78314a47c81e640c9393f2f86eb9/types/client.d.ts#L34
https://github.com/nodejs/undici/blob/ffc6341559ae78314a47c81e640c9393f2f86eb9/types/pool.d.ts#L16

Reply:
This can now be set at the Pool instantiation.